OPHD/UI/PopulationPanel.cpp

```diff
@@ -132,9 +138,13 @@ void PopulationPanel::update()
 
 		const auto roleCount = std::to_string(personCount);
 		renderer.drawText(mFont, personRole + ": ", position + textOffset);
+		std::string roleCountString = roleCount;
+		if (personRole == "Workers")
+			roleCountString = roleCount + " : " + std::to_string(mPopulationPool->availableWorkers());//Add population of workers Available
```

Comment:
I'm somewhat bothered by the use of an `if` statement like this inside of a loop. Maybe we should look at adjusting the loop data, rather than adding `if` statements to the loop body. Maybe we add the extra info to the fields in the defined array, or add more rows to the array. Or maybe we add a new section to that part of the UI.

Reply:
Commit cb642f8 moved the appended portion of the string to the array of tuples eliminating the if statements in the loop.